### DISCUSSION TITLE
用xmake怎么生成Universal macOS Binary？

### DISCUSSION THREAD

**Discussion by wulijie1** [xmake-io/xmake#3709] [General]
如题，cmake里好像这么设一下就行了，set(CMAKE_OSX_ARCHITECTURES "$(ARCHS_STANDARD_64_BIT)")，xmake里要怎么设置呢？

**Comment by waruqi** [comment]
```
xmake macro package -p iphoneos -a "arm64,x86_64"
```

然后到 build 目录递归搜下 universal 

https://xmake.io/#/zh-cn/plugin/builtin_plugins?id=%e5%86%85%e7%bd%ae%e7%9a%84%e5%ae%8f%e8%84%9a%e6%9c%ac
